gulp/util/karma-config.js

```diff
@@ -54,12 +54,15 @@ module.exports = function createConfig (project) {
                 { type: "text" },
             ],
             watermarks: {
-                lines: [80, 90],
-                statements: [80, 90],
+                lines: [79, 90],
+                statements: [79, 90],
             },
         },
         files: filesToInclude,
         frameworks: ["mocha", "chai", "phantomjs-shim", "sinon"],
+        mime: {
```

Comment:
it was added in #409, then lost when in #381 (notice the smaller number 😄), and now restored here.